adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -252,111 +253,423 @@ a@ Constructs a [code]#depends_on_all_leaves# property instance.
 
 |====
 
-=== Graph
+[[sec:khr-graph-command-graph]]
+== Command-graph class
+
+The [code]#command_graph# class represents a directed acyclic graph of nodes,
+where each node represents a single command with zero or more dependencies
```

Comment:
It could also be an empty node or subgraph, in that case the node doesn't represent a command or represents multiple commands.
```suggestion
where each node represents zero or more commands with zero or more dependencies
```

Reply:
@reble Do you have any thoughts on this since you raised this question? Given the discussion I would still be in favour of the "single command" or "single operation" wording here.

Reply:
Consistency is my main concern here. In the node section we define an empty node as:

> An empty node contains no commands.

If a command can consist of zero or more operations, we should update the description.

Reply:
A command is defined as part of the base SYCL spec already so I don't know that we can augment its definition in that way. I'll just go with the originally suggested wording.

Reply:
Yes I was not questioning that part, but our empty node description needs to be updated.
Can do that as a follow up.